docs/lib/analytics/fragments/configure.md

```diff
@@ -0,0 +1,20 @@
+Existing Amazon Pinpoint resources can be used with AWS Amplify through the `awsconfiguration.json` file.
```

Comment:
```suggestion
Existing Amazon Pinpoint resources can be used with AWS Amplify through by including an `awsconfiguration.json` file. Note that this is ___not___ the `amplifyconfiguration.json` file.
```

Reply:
Actually, it is the `amplifyconfiguration.json` file. I made a typo.